server/filters/get-days.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import moment from 'moment';
 
-export default function formatDate(date: Date): string {
+export default function getDays(date: Date): string {
```

Comment:
I think there are different requirements. The designs show 'X days ago' for transporters/promos, but the actual date on article pages.

@Norvard is that correct or can we use the same format everywhere?